model/src/test/kotlin/config/RepositoryConfigurationTest.kt

```diff
@@ -45,6 +45,19 @@ class RepositoryConfigurationTest : WordSpec() {
                 repositoryConfiguration.excludes shouldBe null
             }
 
+            "deserialize to a path regex working with escaped dots" {
```

Comment:
This should have gone to a separate commit, or?

Reply:
Shall I split it up?

Reply:
Not necessarily. I was just confused because the commit message is only about the docs, which is the smaller of the two changes. I'd rather have expected a commit message like:

    model: Add a test for deserializing a repo config with escaped dots in the path

    Fix a wrong configuration example along the way.